### DISCUSSION TITLE
WinUI 3 Gallery now has a icon browser

### DISCUSSION THREAD

**Discussion by JasonWei512** [character-map-uwp/Character-Map-UWP#244] [General]
https://www.microsoft.com/store/productId/9P3JFPWWDZRC

![image](https://user-images.githubusercontent.com/16899918/226072918-9ee0e082-f7ee-4eca-ba57-112c225eccc6.png)

**Comment by ray1997** [comment]
Neat! Well not so much for custom font icons though?
